pkg/agent/dotnetspy/session.go

```diff
@@ -89,7 +89,7 @@ func (s *session) start() error {
 				for k, v := range p.Samples() {
 					s.ch <- line{
 						name: []byte(k),
-						val:  int(v.Milliseconds()) / 10,
+						val:  int(v.Milliseconds()),
```

Comment:
The `examples/dotnet/web` was not working and I found that this is due to wrong calculations here.
I wonder why we were doing `/10` ? For `v.Milliseconds() == 1`, this was resolving to `0` and no samples were reported in stacktrace.

Reply:
It's a leftover. Thank you for fixing this!